example/main.dart

```diff
@@ -33,6 +33,12 @@ class MyApp extends StatelessWidget {
                     child: const Text('Check Update'),
                     onPressed: () => siren.promptUpdate(context),
                   )
```

Comment:
```suggestion
                  ),
```
We have a typo here, a comma is missing.